content/docs/composition-vs-inheritance.md

```diff
@@ -28,7 +28,7 @@ function FancyBorder(props) {
 }
 ```
 
-This lets other components pass arbitrary children to them by nesting the JSX:
+Ini memungkinkan komponen lain untuk menambahkan komponen anak apapun dengan menyarangkan JSX seperti ini:
```

Comment:
Kurang _sreg_ dengan `menyarangkan`. Cari kata penggantinya pun nggak nemu 😅

Reply:
Actually I borrowed `bersarang` and `menyarangkan` as literal translation to `nested` because I saw many Indonesian programming journals translating as that: https://scholar.google.co.id/scholar?hl=en&as_sdt=0%2C5&q=perulangan+bersarang&btnG=

But maybe there's a better translation for it? 😄

Reply:
Ah, okay then.